micropython/ssa/decorators.py

```
@@ -1,37 +1,51 @@
         with the specified property name.
+        If the property has an event associated with it, the event will be checked and published
+        (if triggered) each cycle after the property is published.
 
         The decorated function should return the value to be published.
         The decorated function signature should be:
             async def func() -> Any
 
         Note: See ssa_main decorator documentation for example usage
     """
-    def decorator(func):
+    def decorator(func: Callable[[], Any]):
```

Ruff (0.8.2)

Undefined name `Callable`

(F821)